app/middleware/stripe-setup/check-bank-details-not-submitted.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Local dependencies
+const ConnectorClient = require('../../services/clients/connector_client').ConnectorClient
+const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+const { renderErrorView } = require('../../utils/response')
+const paths = require('../../paths')
+
+module.exports = (req, res, next) => {
+  if (!req.account) {
+    renderErrorView(req, res, 'Internal server error')
```

Comment:
As this is just a generic error can it not be handled by middleware further down the stack, something like 

```js
next(new Error('Invalid Stripe account provided')) 
```

Throwing generic errors within middleware might be hard to track down in the future.

Reply:
Yes, I agree, as discussed we do not have `Error` handling middleware and we use https://github.com/alphagov/pay-selfservice/blob/a90957ad2045b3d124cab59e6572608927f4df89/app/utils/response.js#L49 in a lot of places, so we decided to use the current way and refactor generally at some point